Fixed safari tab bug

bogdan-bodnaruk opened the pull request:
Tab is fixed, but we still have a limitation on Mac (Safari and FF for Mac).
Taking into account that Apple decided that we don't need to be able to select something else except text field, we are not able to focus the user on the Submit button. Anyway, there is a simple way how to solve it. The user should just turn on this feature on his browser. It's weird, but it's true. Here is proof link http://stackoverflow.com/a/2862362

@jmacd3 @QuietOmen

bogdan-bodnaruk committed:
Fixed safari tab bug

rsahlstrom commented:
Yes, if the user wants to tab to the submit button they need to enable that on MacOS. Not something we can handle.

Commit:
Fixing safari tab issue without browser workaround

jmacd3 reviewed (commented):
@janiv We are not trying to submit the form with tab. We just want to be able to move between form fields. We want Enter to submit the form which seems to be the default functionality.

Commit:
Adds focus wrapper

Added mechanism for passing ref in as prop so that we can select the input field when the user focuses on the field. When the user is tabbing we essentially forward the focus down to the input.

BryceHayden reviewed (approved):
It looks good to me.

Commit:
Bump minor version
